### PR TITLE
chore(issue-template): improve correctness, privacy, and comprehensiveness

### PR DESCRIPTION
**Pull request by Rudxain**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 1]
```diff
@@ -27,40 +35,47 @@ body:
...
-  id: os
+  id: kernel # technically accurate
   attributes:
     label: Operating System
     options:
       - Linux
-      - MacOS
-      - Windows
+      - MacOS # Darwin+XNU
+      - Windows # NT
```
Comment (AnonymousWP) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:44]: I personally don't really see the relevance of this. Nobody runs Windows pre-NT. Maybe it's better to ask for the version instead. 😄

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/1-bug-report.yml (hunk 1 above)
Comment (Rudxain) [reply .github/ISSUE_TEMPLATE/1-bug-report.yml:44]: I meant [the kernel](https://en.wikipedia.org/wiki/Windows_kernel) (also part of newer versions, including W11)

[MS is bad at naming](https://www.xda-developers.com/microsoft-remote-desktop-new-name) 💀

Comment (AnonymousWP) [reply .github/ISSUE_TEMPLATE/1-bug-report.yml:44]: Yeah I know you do, but it's better to mention OS version.

Comment (Rudxain) [reply .github/ISSUE_TEMPLATE/1-bug-report.yml:44]: I guess? But it's just a comment, users won't see it

**Review by AnonymousWP — APPROVED** [review]

**Pull request merged**